website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
```

Comment:
A brief comment about what additional features your command offers over the previous counting command would be nice.